### ISSUE TITLE
No Disk Error on Start-Up

### ISSUE THREAD

**Issue by jendrikseipp** [jendrikseipp/rednotebook#220]
On start-up, I receive a pop-up error, "There is no disk in drive \Device\Harddisk1\DR1"

After clicking Continue or the red X multiple times the program runs normally, however the pop-up is impossible to close.
Jack Fowler <email address hidden>

Red Notebook V. 1.1.2

Windows Vista Home Premium SP 2

Config File:

checkForNewVersion=0
closeToTray=0
cloudIgnoreList=comma, filter, separated, these, words
cloudIncludeList=job, mtv, play, spam, work
cloudTabActive=0
dataDir=C:\Users\Jack\.rednotebook\data
dateTimeString=%A, %x %X
leftDividerPosition=282
mainFontSize=-1
mainFrameHeight=752
mainFrameMaximized=0
mainFrameWidth=1024
rightDividerPosition=500
spellcheck=0
userDir=
weekNumbers=0

Logfile

INFO     Writing log to file "C:\Users\Jack\.rednotebook\rednotebook.log"
INFO     Set default locale: "en_US"
INFO     Setting LANG to en_US
INFO     Module glade found
INFO     'module' object has no attribute 'bindtextdomain'
INFO     Chardet guesses {'confidence': 1.0, 'encoding': 'ascii'} for C:\Users\Jack\.rednotebook\configuration.cfg
INFO     Running in portable mode: False
INFO     First Start: False
INFO     RedNotebook version: 1.1.2
INFO     System info: machine: x86, platform: Windows-Vista-6.0.6002-SP2, processor: x86 Family 6 Model 15 Stepping 6, GenuineIntel, python_version: 2.6.4, release: Vista, system: Windows, GTK version: (2, 20, 0), PyGTK version: (2, 16, 0), Yaml version: 3.09
INFO     Using webkit for previews: True
INFO     Cloud ignore list: ['comma', 'filter', 'separated', 'these', 'words']
INFO     Cloud include list: ['job', 'mtv', 'play', 'spam', 'work']
INFO     Using pywebkitgtk for the clouds.
INFO     Using zeitgeist: False
INFO     Opening journal at C:\Users\Jack\.rednotebook\data
INFO     Connected Signals: 2 22 15 
ion: (2, 16, 0), Yaml version: 3.09
2011-02-11 10:17:47,565 INFO     Using webkit for previews: True
2011-02-11 10:19:13,069 INFO     Cloud ignore list: ['comma', 'filter', 'separated', 'these', 'words']
2011-02-11 10:19:13,071 INFO     Cloud include list: ['job', 'mtv', 'play', 'spam', 'work']
2011-02-11 10:19:13,071 INFO     Using pywebkitgtk for the clouds.
2011-02-11 10:19:13,108 DEBUG    Tray icon visible: False
2011-02-11 10:19:13,111 INFO     Using zeitgeist: False
2011-02-11 10:19:13,111 INFO     Opening journal at C:\Users\Jack\.rednotebook\data
2011-02-11 10:19:13,111 DEBUG    Starting to load files in dir "C:\Users\Jack\.rednotebook\data"
2011-02-11 10:19:13,111 DEBUG    Start loading file "C:\Users\Jack\.rednotebook\data\2011-02.txt"
2011-02-11 10:19:13,115 DEBUG    Finished loading file "C:\Users\Jack\.rednotebook\data\2011-02.txt"
2011-02-11 10:19:13,115 DEBUG    Finished loading files in dir "C:\Users\Jack\.rednotebook\data"
2011-02-11 10:19:13,151 DEBUG    Update the cloud (Type: word, Force: True)
2011-02-11 10:19:13,157 DEBUG    Retrieved WordCountDict. Length: 303
2011-02-11 10:19:13,157 DEBUG    Turning the word_count_dict into html
2011-02-11 10:19:13,157 DEBUG    Length word_count_dict: 303
2011-02-11 10:19:13,158 DEBUG    Filtered short words. Length word_count_dict: 164
2011-02-11 10:19:13,158 DEBUG    Filtered blacklist words. Length word_count_dict: 164
2011-02-11 10:19:13,158 DEBUG    Selected most frequent words. Length CloudWords: 42
2011-02-11 10:19:13,158 DEBUG    Min word count: 2, Max word count: 15
2011-02-11 10:19:13,160 DEBUG    Sorted cloud words. Length CloudWords: 42
2011-02-11 10:19:13,160 DEBUG    42 cloud words found
2011-02-11 10:19:13,161 DEBUG    Cloud updated
2011-02-11 10:19:13,164 INFO     Connected Signals: 2 22 15 
2011-02-11 10:19:13,164 DEBUG    Start took 86.0009999275 seconds
2011-02-11 10:19:13,164 DEBUG    Trying to enter the gtk main loop

**Comment by jendrikseipp** [comment]
*Original comment by* **jendrikseipp** (2011-03-03T18:18:38Z):

----------------------------------------

Sorry, I have never seen anything similar. Is the problem still there with version 1.1.3?

**Comment by jendrikseipp** [comment]
*Original comment by* **jafowler** (2011-03-03T19:33:33Z):

----------------------------------------

No, the problem seems to have resolved itself.  I only got the error 
after initial installation, but now it's not happening.  I did nothing 
to correct it - it just stopped.

Thanks for your help,

Jack Fowler (Greg to some)
Herbalife Distributor
757-802-3705
www.herbalhealthdirect.com/stay_well
